dome9/data_source_dome9_continuous_compliance_finding_test.go

```diff
@@ -0,0 +1,36 @@
+package dome9
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+
+	"github.com/terraform-providers/terraform-provider-dome9/dome9/common/resourcetype"
+	"github.com/terraform-providers/terraform-provider-dome9/dome9/common/testing/method"
+)
+
+func TestAccDataSourceFindingSearchBasic(t *testing.T) {
```

Comment:
I would try to create in Terraform the new asset you created to check that everything really works as required. Because it is possible that the test here misses cases.